bin/node/browser-testing/Cargo.toml

```diff
@@ -1,24 +1,20 @@
 [package]
 name = "node-browser-testing"
-version = "3.0.0-dev"
+version = "4.0.0-dev"
```

Comment:
Why are major versions bumped?

Reply:
Yeah maybe that's my bad. I was aiming for consistency of everything being either v 4.0.0 or 0.10.0. but talking with Ben it seems that we're heading towards every crate having its own independent version number.